src/libvfcfuncinstrument/libVFCFuncInstrument.cpp

```diff
@@ -221,12 +314,56 @@ void InstrumentFunction(std::vector<Value *> MetaData,
   std::vector<Value *> ExitArgs = OutputMetaData;
```

Comment:
Can you please add some comments to step 6, to describe what each block of code does ?

Reply:
Btw, the code is quite explicit and clean. Congrats !

Reply:
I will add more comment for steps 2 and 6